src/add-game/index.md

```diff
@@ -0,0 +1,83 @@
+# How do I add my game to Game Jolt?
+
+First of all, you need to have a developer account. If you don't yet have one, just [go here](http://gamejolt.com/dashboard/become-developer/). 
+
+1. If you don't have a Game Jolt account at all, [sign up](http://gamejolt.com/auth/sign_up/) for one, and choose "Developer" when the time comes. 
+2. Dev account all set up? Good. Select "Dashboard" on the black nav bar at the top of the page. Welcome to your [developer dashboard](http://gamejolt.com/dashboard/)! Now hit the big "Add a Game" button. Go ahead, do it. 
+
+## Step 1: Add game information
+
+You'll need to fill in some basic information about your game in order to proceed.
+
+### Title
+
+Surely your game has a name. Put it here.
+
+[Choosing a game title](/game-title/index.md)
+
+### Website 
+
+If you wish to link to a personal website about the game or about your games in general, this is the place to do it.
+
+### Category
+
+You have to choose a category for your game. The category you pick is entirely up to you. Categories are meant to be helpful, not restrictive. They are signposts to aid gamers in finding the games they want to play.
+
+[Choosing a game category](/game-category/index.md)
+
+### Development Status
+
+You must indicate the status of your game's development. You can choose one of three options:
+
+- Finished: Your game is complete. You're uploading a build, and that build is stable. You might upload fixes or other small updates, but you don't expect to make any major changes. 
+- Work-in-Progress: Your game is still in development. You might have an alpha/beta/test build for people to play, but it may be unstable. Major updates are planned. 
+- Canceled: Your game is not what you'd consider complete, but you've ceased development on it. There is a playable build (stable or not), but you don't plan to ever update the game. 
+
```

Comment:
Should we cut out the description of the statuses and link to the help page instead?